### DISCUSSION TITLE
I have a question about the sendTransaction

### DISCUSSION THREAD

**Discussion by b0jun** [Web3Auth/Web3Auth#1304] [Help]
I didn't do the eth sign, how can I sendTransaction it right away?
When was the eth sign done?

sample code
```
    const sendTransaction = async () => {
        try {
            const web3 = new Web3(provider as any);
            const accounts = await web3.eth.getAccounts();
            const txRes = await web3.eth.sendTransaction({
                from,
                to,
                value: web3.utils.toWei('0.01'),
            });
        } catch (error) {
            console.log('error', error);
        }
    };

**Comment by ieow** [comment]
The web3auth `provider` that passed in to Web3 has the key.
when you send Transaction
```
            const txRes = await web3.eth.sendTransaction({
                from,
                to,
                value: web3.utils.toWei('0.01'),
            });
```

`provider` will sign it and forward it to the network

**Reply by JHKBO** [reply]
Hi, ieow, 

I have one more question following your answer like below.

Initially, we would like to know whether the function of sendTransaction in web3auth does not need to require "sign step" in the source during below case.
from wallet address: "someone" (not me)
to wallet address: "another someone"(not me)

Please kindly let me know.

**Reply by yashovardhan** [reply]
The Web3Auth provider is the standard EIP155 provider. You cannot send transaction on someone else's behalf unless it is signed. This function automatically signs the transaction before sending upon your behalf but cannot do it on another person's behalf.
